benchmarks/scripts.py

```diff
@@ -67,9 +75,12 @@ def circuit_benchmark(nqubits, backend, circuit_name, circuit_options=None,
             result = result.numpy()
         transfer_times.append(time.time() - start_time)
         del(result)
+        del(circuit)

```

Comment:
Stupid remark: `del` is a statement (like `import`), while here it is used like a function.

Empirically, this is allowed (not working for `import` instead), but I don't even find it documented on https://docs.python.org...
```suggestion
        del circuit
```

Reply:
indeed, pycodestyle was telling me the same :) shall I also change the `del(result)`s?